tests/directive.spec.ts

```diff
@@ -44,112 +42,6 @@ function buildConstDirectiveNodes(name: string, args: Record<string, string>): C
 }
 
 describe('format directive config', () => {
-  describe('formatDirectiveObjectArguments', () => {
-    const cases: {
-      name: string
-      arg: DirectiveObjectArguments
-      want: FormattedDirectiveObjectArguments
-    }[] = [
-      {
-        name: 'normal',
-        arg: {
-          uri: 'url',
-          email: 'email',
-        },
-        want: {
-          uri: ['url', '$2'],
-          email: ['email', '$2'],
-        },
-      },
-      {
-        name: 'contains array',
-        arg: {
-          startWith: ['matches', '/^$2/'],
-          email: 'email',
-        },
-        want: {
-          startWith: ['matches', '/^$2/'],
-          email: ['email', '$2'],
-        },
-      },
```

Comment:
I think it is important to maintain backward compatibility. Where have these test cases gone? I would like to see them restored if possible.